frontends/web/src/components/aopp/vasp.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2021 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { h, JSX, RenderableProps } from 'preact';
+import * as styles from './vasp.css';
+import TwentyOneAnalyticsLogo from '../../assets/exchanges/logos/21analytics.png';
+import BitcoinSuisseLogo from '../../assets/exchanges/logos/bitcoin_suisse.png';
+import BittrLogo from '../../assets/exchanges/logos/bittr.png';
+import BityLogo from '../../assets/exchanges/logos/bity.png';
+import PocketBitcoinLogo from '../../assets/exchanges/logos/pocketbitcoin.svg';
+
+interface VASPProps {
+    fallback?: JSX.Element;
+    hostname: string;
+    prominent?: boolean;
+    withLogoText?: string;
+}
+
+const VASPLogoMap = {
```

Comment:
there are entries with subdomains and entries without subdomains.

it sounds more robust to discard subdomains and only go for the main part?

e.g. bitcoinsuisse.com redirects to https://www.bitcoinsuisse.com/ (including www)

Reply:
> there are entries with subdomains and entries without subdomains.

this is the reason this map exsits, so we could map `api.host.tld` to `host.tld` or remove the `testing.` subdomain.

Reply:
> it sounds more robust to discard subdomains and only go for the main part?

Do you mean for all domains or just the ones we know?

Noob me doesn't know how to remove the subdomain i.e. the TLD could be .com.au or any other weird stuff.

Reply:
ok let's leave it then